src/Symfony/Component/Validator/Constraints/TimestampValidator.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+use Symfony\Component\Validator\Exception\UnexpectedTypeException;
+use Symfony\Component\Validator\Exception\UnexpectedValueException;
+
+class TimestampValidator extends ConstraintValidator
+{
+
+    public function validate(mixed $value, Constraint $constraint): void
+    {
+        if (!$constraint instanceof Timestamp){
+            throw new UnexpectedTypeException($constraint::class, self::class);
+        }
+
+        if (null === $value){
+            return;
+        }
+
+        if (!is_int($value)){
+            throw new UnexpectedValueException($value, 'int');
+        }
+
+        if (0 > $value){
```

Comment:
This limitation is arbitrary. A negative value is a valid timestamp and so is zero.